doc/content/toolstack/high-level/daemons.md

```diff
@@ -0,0 +1,52 @@
++++
+title = "Daemons"
+weight = 20
++++
+
+The Toolstack consists of a set of co-operating daemons:
+
+xapi
+: manages clusters of hosts, co-ordinating access to shared storage and networking.
+
+xenops
```

Comment:
I think it should be named as xenopsd, or refer to the full name, xenopsd-xc

Reply:
Yes, typo!